sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/client.py

```diff
@@ -713,7 +687,7 @@ def list_certificate_versions(self, name: str, **kwargs: "**Any") -> AsyncIterab
         This operation requires the certificates/managecontacts permission.
 
         :return: Contacts
-        :rtype: Iterator[~azure.security.certificates._models.Contact]
+        :rtype: Iterator[~azure.keyvault.certificates.models.Contact]
```

Comment:
I think for doc purposes it's better to be specific here--we actually return a `list`--rather than claim to return a Protocol.

Reply:
ok makes sense, I'll switch instances of Iterator to list. Thanks!

Reply:
Iterator for Contacts to be specific because the rest are ItemPaged